src/schema/v2/notifications/index.ts

```diff
@@ -56,6 +57,10 @@ export const NotificationType = new GraphQLObjectType<any, ResolverContext>({
       ...date(({ date }) => date),
       deprecationReason: "Please use `publishedAt` instead",
     },
+    content: {
```

Comment:
```graphql
notification(id) {
  items {
    ... on ViewingRoomPublishedNotificationItem {
      ...
    }
  }
}
```

having the field be plural feels more flexible, since once can and will likely return multiple types. But at the same time it could also feel a bit confusing since one might assume a collection 🤷

Reply:
I'm not totally sold on this! It feels as though returning a single 'thing' that represents a single feed item (even though the feed item itself can be an aggregation of multiple 'activities' during a 24 hour period), is what we want.